### DISCUSSION TITLE
Turn off iPad display after setting up SEB?

### DISCUSSION THREAD

**Discussion by philpav** [SafeExamBrowser/seb-mac#245] [Q&A]
Hi, 
we want to hand out preconfigured iPads to students. But as soon as we load the configuration in SEB it's not possible to turn off the display. 
We want to prepare the iPads one or two days before the exam and don't want to keep the display on the whole time. 
We also don't want the students to be able to quit SEB or shutdown the iPads themselves. Is it possible somehow?

Thank you!

**Comment by danschlet** [comment]
I don't understand when do you want to prevent the students to quit SEB or shutdown the iPads themselves, from the moment they receive the iPads (one or two days before the exam) or only after the exam started?

The first case is not possible. You cannot lock the iPads into SEB for one or two days and turn off the display at the same time, that's just not possible in the iOS Assessment Mode (AAC and most likely also not with the MDM pre-configured ASAM single app mode). When the single app mode is active, the device cannot be put to sleep or the display turned off, the Apple API just doesn't allow that, there's nothing SEB can do about that.

For the second case you just tell students to start SEB only at the time when the exam starts. You can use a client configuration (which you can deploy using MDM to the iPads) without a quit password set which would load some exam portal page or the home page of your LMS/exam system. If there is no quit password set, SEB doesn't run in the Assessment Mode and the display can be put to sleep. In that client config you can also change the setting in SEB's Settings / Security / Prevent Auto-Lock to disabled, then the display will be switched off automatically as in other apps (but this setting has no effect when the single app/Assessment Mode is active).

Then on the portal page or the LMS students navigate to the correct exam and there can be another sebs:// link to an SEB exam config file (maybe your LMS/exam system offers an SEB integration which generates exam settings automatically, like for example Moodle), which has a quit password set and therefore runs in the single app mode. Then they cannot quit SEB or shutdown the iPads during the exam.

Please note that there is a key shortcut to restart iPads even when they are running in the single app/Assessment Mode. If you want to prevent even this, you need to use the MDM preconfigured ASAM single app mode (only possible on supervised iPads), as in that mode SEB is automatically restarted after a forced restart of the device.
